test/run.py

```diff
@@ -39,7 +34,7 @@ async def main():
     if script is None:
         raise Exception("unrecognized scenario", scenario)
 
-    drop = ffi.Drop(lib, ffi.KeysCtx(runner))
+    drop = ffi.Drop(ffi.KeysCtx(runner))
```

Comment:
Where is the entrypoint where I could track and see how the lib.so is being loaded and used in UniFFI generated code? Basically from my POV we don't pass the library file - how does UniFFI know where to look for it?

Reply:
the python bindings load the library and perform all of the setup on import

Reply:
I can only add that it loads the `.so` from the directory where the bindings code file is located. I've learned this the hard way

Reply:
It does this:
```python
    if sys.platform == "darwin":
        libname = "lib{}.dylib"
    elif sys.platform.startswith("win"):
        # As of python3.8, ctypes does not seem to search $PATH when loading DLLs.
        # We could use `os.add_dll_directory` to configure the search path, but
        # it doesn't feel right to mess with application-wide settings. Let's
        # assume that the `.dll` is next to the `.py` file and load by full path.
        libname = os.path.join(
            os.path.dirname(__file__),
            "{}.dll",
        )
    else:
        # Anything else must be an ELF platform - Linux, *BSD, Solaris/illumos
        libname = "lib{}.so"

    libname = libname.format("norddrop")
    path = os.path.join(os.path.dirname(__file__), libname)
    lib = ctypes.cdll.LoadLibrary(path)
    return lib
```

Reply:
:bowing_man: